I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 580, in script
    with fastdissolve
  File "game/script.rpy", line 580, in <module>
    with fastdissolve
NameError: name 'fastdissolve' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 580, in script
    with fastdissolve
  File "C:\Users\Admin\Desktop\Folders\Programs\Renpy\renpy\ast.py", line 1259, in execute
    trans = renpy.python.py_eval(self.expr)
  File "C:\Users\Admin\Desktop\Folders\Programs\Renpy\renpy\python.py", line 1606, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\Admin\Desktop\Folders\Programs\Renpy\renpy\python.py", line 1601, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 580, in <module>
    with fastdissolve
NameError: name 'fastdissolve' is not defined

Windows-8-6.2.9200
Ren'Py 6.99.10.1227
Water Girl 0.0
